library/src/main/java/com/novoda/downloadmanager/lib/StorageManager.java

```diff
@@ -294,7 +294,7 @@ File getDownloadDataDirectory() {
     }
 
     public static File getDownloadDataDirectory(Context context) {
-        return context.getCacheDir();
+        return new File(context.getCacheDir().toString() + "/download-manager/");
```

Comment:
could also use `File.separator` but I suppose we're pretty safe with the environment :wink: